fix(common): Added expected homeURL that was missing

PolarisTLX opened the pull request:
<!-- freeCodeCamp Pull Request Template -->

<!-- IMPORTANT Please review https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md for detailed contributing guidelines -->
<!-- Help with PRs can be found at https://gitter.im/FreeCodeCamp/Contributors -->
<!-- Make sure that your PR is not a duplicate -->

#### Pre-Submission Checklist
<!-- Go over all points below, and after creating the PR, tick all the checkboxes that apply. -->
<!-- All points should be verified, otherwise, read the CONTRIBUTING guidelines from above-->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change
<!-- What type of change does your code introduce? After creating the PR, tick the checkboxes that apply. -->
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:
<!-- Go over all points below, and after creating the PR, tick the checkboxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask in the Contributors room linked above. We're here to help! -->
- [ ] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #16836

#### Description
<!-- Describe your changes in detail -->
Fixes a broken link, where link is stored as a variable "homeURL" that is never defined.

Added variable "homeURL" into the file "common/utils/constantStrings.json" 
that is required to be imported into file Timeline.jsx

Could not test properly on local machine.

By Paul (@PolarisTLX) and Kyle (@jklemon17)

PolarisTLX committed:
fix(common): Added expected homeURL that was missing

added variable homeURL that is imported into file Timeline.jsx

BREAKING CHANGE: none

Closes #16836

Bouncey commented on common/utils/constantStrings.json:
```diff
@@ -4,5 +4,6 @@
   "githubUrl": "https://github.com/freecodecamp/freecodecamp",
-  "RSA": "https://forum.freecodecamp.org/t/the-read-search-ask-methodology-for-getting-unstuck/137307"
+  "RSA": "https://forum.freecodecamp.org/t/the-read-search-ask-methodology-for-getting-unstuck/137307",
+  "homeURL": "https://www.freecodecamp.org/challenges/learn-how-freecodecamp-works"
```
The home url is `https://freecodecamp.org`

PolarisTLX replied:
Alright I will update the link held in the variable, and that should update the pull request for approval.

PolarisTLX committed:
Changed the homeURL variable link

Changed the homeURL variable link to be "https://freecodecamp.org"
Instead of  "https://www.freecodecamp.org/challenges/learn-how-freecodecamp-works"
as per @Bouncey's requested change.

Bouncey commented:
Thanks @PolarisTLX  and @jklemon17 :+1:
